doc/deprecating-badges.md

```diff
@@ -90,13 +90,17 @@ Past that point, all related code will be deleted, and a not found error will be
 
 Here is a listing of all deleted badges that were once part of the Shields.io service:
 
+- Beerpay
+- Bintray
 - bitHound
 - Cauditor
 - CocoaPods Apps
 - CocoaPods Downloads
 - Codetally
 - continuousphp
 - Coverity
+- David
+- dependabot
 - Dockbit
 - Dotnet Status
```

Comment:
It's a bit tricky for me to determine whether I think it's worth mentioning the Docker build service here :thinking: 

I believe what you've got here is probably for the best, since it could be more confusing than helpful to have it listed (i.e. we still have a "Docker Build" badge from the end user pov, the deprecated one just had the nuances of some legacy with the platform and an associated badge route)

Reply:
I figured the potential for confusion probably outweighed the benefits, but anyway I'm not convinced this list is read through very often. Let's merge as is :)